angular/projects/admin/src/app/admin/settings/settings-dialog-manage-languages/settings-dialog-manage-languages.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col">
+  </div>
+</div>
+<div class="row">
+  <div class="col">
+    <form [formGroup]="languagesForm"
+          novalidate>
+      <div formArrayName="languages">
+        <div class="row"
+             *ngFor="let language of languagesFormArray.controls; let i=index"
+             [formGroupName]="i">
+          <div class="col">
+            <mat-form-field>
+              <input matInput
+                     placeholder="Language"
+                     formControlName="name">
+            </mat-form-field>
+            <button mat-icon-button
+                    color="warn"
+                    *ngIf="i > 0"
+                    (click)="removeLanguage(i)">
+              <mat-icon aria-label="Example icon-button with a heart icon">delete</mat-icon>
+            </button>
+          </div>
+        </div>
+      </div>
+    </form>
+    <div class="actions">
+      <button mat-raised-button
+              color="primary"
+              (click)="save()">Save</button>
+      <button mat-raised-button
+              color="accent"
+              (click)="addLanguage()">Add Language</button>
```

Comment:
Use `<mat-dialog-actions>`. See: https://material.angular.io/components/dialog/overview#dialog-content

Reply:
Solved on this commit https://github.com/oddbit/tanam/pull/51/commits/0f799bcd17b0da668755699d7cc322d045e11e05